src/models/db/account.js

```diff
@@ -44,10 +25,8 @@ class Account extends Model {
     if (data.balance) {
       data.balance = Number(data.balance)
     }
-    if (data.password) {
-      data.password_hash = hashPassword(data.password)
-      delete data.password
-    }
+
+    delete data.password_hash
```

Comment:
That makes sense. The reason this initially jumped out at me is that we were deleting the hash, but not the password (I didn't see the code in `setAccount`), so I thought we would store the plaintext password in the db.